kt/godot-runtime/src/main/kotlin/godot/core/Functions.kt

```diff
@@ -2,19 +2,85 @@ package godot.core
 
 import godot.util.camelToSnakeCase
 
+enum class PropertyHint {
```

Comment:
I really hate this enum :P Can we drop the `PROPERTY_HINT` prefix?

Reply:
I don't have strong opinion about it, so yes.